Entities/CatEntity.php

```diff
@@ -0,0 +1,7 @@
+<?php
```

Comment:
Why is this unused file in this PR?

Reply:
all the files that will be used in the project were created at the very beginning stage for everyone's convenience

Reply:
That's makes sense!
Don't make things before they're used, please remove them from this PR.

Reply:
ok